services/core-web/common/actionCreators/incidentActionCreator.js

```diff
@@ -69,3 +69,22 @@ export const fetchIncidents = (payload) => (dispatch) => {
     .catch(() => dispatch(error(reducerTypes.GET_INCIDENTS)))
     .finally(() => dispatch(hideLoading()));
 };
+
+export const deleteMineIncident = (mineGuid, incidentGuid) => (dispatch) => {
+  dispatch(request(reducerTypes.DELETE_INCIDENT));
+  return CustomAxios()
```

Comment:
add `dispatch(showLoading());` after line 74 to all actionCreators that have a `.finally(() => dispatch(hideLoading()));`

Reply:
done